graphql/schema.graphqls

```diff
@@ -409,11 +441,6 @@ type Validator {
     """
     country: String!
 
-    """
-    The validator rpc node endpoint.
-    """
-    rpcEndpoint: URI
```

Comment:
Is it intentional to remove this attribute from validator ?

Reply:
Absolutely not! Good catch, certainly accidental during conflict resolution 😿